apps/extension/src/routes/popup/home/inspect-address/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box } from '@penumbra-zone/ui/components/ui/box';
+import { IconInput } from '@penumbra-zone/ui/components/ui/icon-input';
+import { useEffect, useState } from 'react';
+import { MagnifyingGlassIcon } from '@radix-ui/react-icons';
+import { Result as TResult } from './types';
+import { getResultFromBech32mAddress } from './get-result-from-bech32m-address';
+import { Result } from './result';
+
+export const InspectAddress = () => {
+  const [address, setAddress] = useState('');
+  const [result, setResult] = useState<TResult | undefined>();
+
+  useEffect(() => {
+    void getResultFromBech32mAddress(address).then(setResult);
+  }, [address]);
+
+  return (
+    <Box label='Inspect address'>
```

Comment:
Is "inspect" the best word here? I wonder if `Validate Address` would be better.

Reply:
👍🏻 done